src/alexapi/device_platforms/dummy.py

```diff
@@ -11,36 +14,28 @@ def __init__(self, config):
 		self.should_confirm_trigger = False
 
 	def setup(self):
-		if self._pconfig['verbose']:
```

Comment:
Since you got rid of the platform config option _verbose_, delete this:

```
    # whether to print messages when methods are called
    verbose: false
```

from the config template. Be sure to keep the `dummy:`, so the config loads successfully.